src/main/java/com/amihaiemil/zold/UserAgentRequestHeader.java

```diff
@@ -0,0 +1,66 @@
+package com.amihaiemil.zold;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.Properties;
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.protocol.RequestDefaultHeaders;
+import org.apache.http.message.BasicHeader;
+
+/**
+ * User Agent Request Header Interceptor.
+ * 
+ */
+
+final class UserAgentRequestHeader extends RequestDefaultHeaders {
```

Comment:
@ammaratef45 Can you add the ``@author``, ``@version`` and ``@since`` tags? Also, add the copyright header as in the other files. 

Furthermore, leave  a ``@todo`` puzzle for actually using this class when instantiating the HttpClient :)

Reply:
Hmm, for this should I make the author me or the one I copied his work?